docs/data/material/components/material-icons/material-icons.md

```diff
@@ -10,20 +10,23 @@ githubLabel: 'package: icons'
 
 <p class="description">2,100+ ready-to-use React Material Icons from the official website.</p>
 
-The following npm package,
-[@mui/icons-material](https://www.npmjs.com/package/@mui/icons-material),
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+<br/>
```

Comment:
Had to add this `<br/>` element because, for some reason, adding a line here wasn't adding the space 🤔